boxes/boxes/vanilla/src/index.html

```diff
@@ -7,7 +7,7 @@
     <style></style>
   </head>
   <body>
-    <button id="deploy">Deploy</button>
+    <button id="deploy" hidden>Deploy</button>
     <button id="get" hidden>Get Number</button>
```

Comment:
Why make this hidden?

Reply:
Playwright waits on hidden elements when you await a `.click()` call. Since the creation of the account manager is now async, we hide it, wait until initialization, then unhide and the test can progress.